src/instructlab/model/evaluate.py

```diff
@@ -301,6 +301,7 @@ def launch_server(
             logger.warning(
                 f"Based on your hardware configuration, when using vLLM, we recommend setting max-workers between {recommended_min_workers} and {recommended_max_workers} for optimal performance"
             )
+        gpus = gpus or eval_serve.vllm.gpus
```

Comment:
Shouldn't this be achieved with:

```
    cls=clickext.ConfigOption,
    config_sections="vllm",
```

...in the config option definition? Like it's done for the `serve` `--gpus` CLI option.

Reply:
The `evaluate` section of the config does not have a `vllm` section. See the comment below on how we need to go to the drawing board to figure out what the common interface under the hood/in the config should be across commands for `--gpus`.

Reply:
Oh gosh!